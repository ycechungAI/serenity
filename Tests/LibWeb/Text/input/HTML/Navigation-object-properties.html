<script src="../include.js"></script>
<script>
    test(() => {
        let n = window.navigation;

        // FIXME: Once we set up the interaction between Navigables and Navigation,
        //        These two should become 1 and [object NavigationHistoryEntry], respectively
        println(`entries is empty: ${n.entries().length == 0}`);
        println(`currentEntry is null: ${n.currentEntry == null}`);

        println(`transition is null: ${n.transition == null}`);
        println(`canGoBack: ${n.canGoBack}`);
        println(`canGoForward: ${n.canGoForward}`);       
    });
</script>
