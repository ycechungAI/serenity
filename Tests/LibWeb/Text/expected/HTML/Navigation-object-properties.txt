entries is empty: true
currentEntry is null: true
transition is null: true
canGoBack: false
canGoForward: false
